docs/sub_completion.md

```diff
@@ -0,0 +1,30 @@
+# Submission completion flow
```

Comment:
FYI I already added some flow documentation here: https://github.com/open-formulieren/open-forms/blob/master/docs/developers/submissions.rst

Reply:
this will be deleted later, it was just a reminder/sketching for myself ^^